extension.js

```diff
@@ -1,26 +1,44 @@
 const Gio = imports.gi.Gio;
 const Meta = imports.gi.Meta;
+const Mainloop = imports.mainloop;
 
 const ExtensionUtils = imports.misc.extensionUtils;
 
 const WORKSPACE_COUNT_KEY = 'workspace-count';
 const WORKSPACE_INDEX = 'workspace-index';
 const WALLPAPERS_KEY = 'workspace-wallpapers';
 const BACKGROUND_SCHEMA = 'org.gnome.desktop.background';
-const CURRENT_WALLPAPER_KEY = 'picture-uri';
+let   CURRENT_WALLPAPER_KEY = 'picture-uri';
```

Comment:
Please change to const to be consistent with other variables.

Reply:
CURRENT_WALLPAPER_KEY can't be a constant now, it need to be variable because it has to change based on the gnome scheme color (DARK or LIGHT)